src/platform/plugins/shared/unified_search/public/project_picker/project_picker.tsx

```diff
@@ -0,0 +1,427 @@
+
+  useEffect(() => {
+    // TODO: replace with fetch linked projects from cross project API
+    setProjects(
+      Object.values(response.linked_projects).sort((a, b) => a._alias.localeCompare(b._alias))
+    );
+  }, []);
+
+  const button = (
+    <EuiToolTip
+      delay="long"
+      content={strings.getProjectPickerButtonLabel(projects.length, linkedProjects.length + 1)}
+      disableScreenReaderOutput
+    >
+      <EuiButtonIcon
+        type="link"
+        display="base"
+        iconType="cluster" // TODO: replace with cross project icon when available in EUI
+        aria-label={strings.getProjectPickerButtonAriaLabel()}
+        data-test-subj="addFilter"
+        onClick={() => setShowProjectPickerPopover(!showProjectPickerPopover)}
+        size="s"
+        css={{
+          backgroundColor: euiTheme.colors.backgroundBaseFormsPrepend,
+          border: `${euiTheme.border.width.thin} solid ${euiTheme.colors.borderBasePlain}`,
+        }}
+      />
+    </EuiToolTip>
+  );
+
+  const closePopover = () => setShowProjectPickerPopover(false);
+
+  const renderProjectsList = () =>
+    projects.map((project, index) => {
+      const tags = Object.entries(project)
+        .filter(([key]) => !key.startsWith('_'))
+        .map(([key, value]) => `${key}: ${value}`);
+
+      return (
+        <EuiFlexItem
+          key={project._id}
+          css={{
+            borderTop:
+              index > 0
+                ? `${euiTheme.border.width.thin} solid ${euiTheme.colors.borderBaseSubdued}`
+                : undefined,
+            padding: `${euiTheme.size.s}`,
+          }}
+        >
+          <EuiFlexGroup responsive={false} alignItems="center" justifyContent="spaceBetween">
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup responsive={false} gutterSize="s">
+                <EuiFlexItem grow={false}>
+                  <EuiIcon type={getSolutionIcon(project._type)} />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiText size="s" color="text">
+                    {project._alias || project._id}
+                  </EuiText>
+                </EuiFlexItem>
+                {project._id === originProject._id ? (
+                  <EuiFlexItem grow={false}>
+                    <EuiIconTip size="m" type="flag" content={strings.getOriginProjectLabel()} />
+                  </EuiFlexItem>
+                ) : null}
+              </EuiFlexGroup>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup responsive={false} gutterSize="s" alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiText size="s" color="subdued">
+                    {`${getCSPLabel(project._csp)}, ${project._region}`}
+                  </EuiText>
+                </EuiFlexItem>
+                {tags.length ? (
+                  <EuiFlexItem grow={false}>
+                    <EuiToolTip
```

Comment:
a11y: When tabbing through the picker, this element should be included in the tab order. We can achieve this by adding tabIndex={0}.